lib/internal/modules/esm/loader.js

```diff
@@ -165,20 +166,28 @@ class ESMLoader {
         __proto__: null,
         ...importAssertions,
       };
+      // We need a second clone in case the first one gets mutated.
+      originalImportAssertions = {
+        __proto__: null,
+        ...importAssertions,
+      };
```

Comment:
The first one can't get mutated by user hooks, so there's no point in cloning it I think
```suggestion
```